Python_Engine/Python/src/python_toolkit/bhom/analytics.py

```diff
@@ -87,7 +87,7 @@ def wrapper(*args, **kwargs) -> Any:
 
                 if ANALYTICS_LOGGER.handlers[0].baseFilename != log_file:
```

Comment:
```suggestion
                if ANALYTICS_LOGGER.handlers[0].baseFilename != str(log_file):
```
If the logger base file name is a string, then it will never be equal to a Path even if they represent the same location, so need to convert the Path to str as well for consistency
![image](https://github.com/user-attachments/assets/7f01a5a9-e744-4f1e-b20f-287771a9ad81)
(pathlib auto converts `/` to `\\` for window)